ddtrace/internal/writer.py

```diff
@@ -39,7 +38,9 @@ def __init__(self, hostname='localhost', port=8126, uds_path=None, https=False,
         self.dogstatsd = dogstatsd
         self.api = api.API(hostname, port, uds_path=uds_path, https=https,
                            priority_sampling=priority_sampler is not None)
-        self._stats_rate_counter = 0
+        self._last_thread_time = 0
```

Comment:
sounds not necessary

Reply:
you mean, don't initialize if `time.thread_time()` is not available?

Reply:
Yes

Reply:
```suggestion
```